vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
Removing it from the assets path works for me.  Added an initializer to `spec/dummy` with this:

```ruby
Rails.configuration.assets.paths.reject! do |path|
  path.match(/rails-assets-react-\d/)
end
```

If we need the addons in any of the other a2g apps, I guess we'd have to add the fix there, too, but we'd have to configure `react-rails` anyway, right?